lib/components/viewers/viewers.css

```diff
@@ -273,3 +273,59 @@ a.routeDetails {
   float: right;
   width: 50px;
 }
+
+/* Route Viewer Updates */
+.search-and-filter {
+  margin-top: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+}
+.search-and-filter select {
+  margin: 0 5px;
```

Comment:
(nitpick) This is probably causing the mode filter selector to not be flush with the right margin.

Reply:
Thank you! That had been bothering me. Fixed in a981abe4dac16acfc816cf54aa4b975ff102118f